bluesky/plans.py

```diff
@@ -2260,6 +2283,7 @@ def tweak(detector, target_field, motor, step, *, md=None):
         md,
         {'detectors': [detector.name],
          'motors': [motor.name],
+         # 'num_steps': 'adaptive',
```

Comment:
I think it's better to leave this out (as your hedged `#` seems to suggest). IMO, it's easier to test `if 'num_steps' in header.start` than `if isinstance(header.start.get('num_steps'), six.string_types)`.